e2e/paths/enterChildren_test.js

```diff
@@ -7,14 +8,21 @@ Before((I, caseViewPage) => {
   caseViewPage.goToNewActions(config.applicationActions.enterChildren);
 });
 
-Scenario('completing half of the enter children in the c110a application', (I, enterChildrenPage) => {
+Scenario('completing half of the enter children in the c110a application', (I, enterChildrenPage, caseViewPage) => {
   enterChildrenPage.enterChildDetails('Timothy', '01', '08', '2015');
   enterChildrenPage.defineChildSituation('01', '11', '2017');
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.enterChildren);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  I.seeAnswerInTab(1, 'Child 1', 'Child\'s full name', 'Timothy');
```

Comment:
I guess would would have helped is if we started using proper apostrophe character instead of single quotation mark. Further improvement though.